clusters_test.py

```diff
@@ -1,106 +1,113 @@
-import sys
+import os
 from collections import Counter
 
 from absl.testing import absltest
 import numpy as np
 import pandas as pd
-import traitlets
 
 import clusters
 
 
 class ClusterTests(absltest.TestCase):
```

Comment:
The rest of the files, and the Google standard, use 2 space for each indent level.  Not sure how this got changed.  Can you put it back to two spaces?